### DISCUSSION TITLE
vc-navigation，使用后，组件的css丢失，导致样式出现问题

### DISCUSSION THREAD

**Discussion by niaonile** [zouyaoji/vue-cesium#326] [Q&A] [answered]
![微信图片_20221208152833](https://user-images.githubusercontent.com/39119253/206385468-e02fdc05-e562-4377-a660-1f71790c736f.png)

**Comment by zouyaoji** [comment, marked as answer]
官网文档是没这种问题的，
如果要定位问题，请提供复现用例哈，谢谢。

**Reply by niaonile** [reply]
我的失误，我居然略过注释css-loader了，多有打扰。
文档写需要css-loader，提示是需要postcss-import这个，有什么区别么
